api_db.php

```diff
@@ -368,15 +368,30 @@ function resolveHostname($clientip, $printIP)
 	// Parse the DB result into graph data, filling in missing interval sections with zero
 	function parseDBData($results, $interval, $from, $until) {
 		$data = array();
+		$first_db_timestamp = -1;
 
 		if(!is_bool($results)) {
 			// Read in the data
 			while($row = $results->fetchArray()) {
 				// $data[timestamp] = value_in_this_interval
 				$data[$row[0]] = intval($row[1]);
+				if($first_db_timestamp === -1)
```

Comment:
Based on line 371, is this test not always true? Maybe I am being blind but `$first_db_timestamp` is always `-1` at this point..?

Reply:
Only on the first iteration of this while loop

Reply:
I guess I didn't look at row 379.... am idiot.